cluster/vagrant/setup_kubernetes_common.sh

```diff
@@ -61,6 +61,14 @@ systemctl enable kubelet && systemctl start kubelet
 # Disable libvirt cgroup management
 echo "cgroup_controllers = [ ]" >> /etc/libvirt/qemu.conf
 
+# Let libvirt listen on TCP for migrations
+echo 'LIBVIRTD_ARGS="--listen"' >> /etc/sysconfig/libvirtd
+echo 'listen_tcp = 1' >> /etc/libvirt/libvirtd.conf
```

Comment:
I'll go with whatever you choose, but this might be more readable:

cat << EOF >>/etc/libvirt/libvirt.conf
listen_tcp = 1
tcp_port = "16509"
auth_tcp = "none"
listen_addr = "0.0.0.0"
listen_tls = 0
EOF